server/src/services/entityService.ts

```diff
@@ -59,11 +59,11 @@ class EntityService {
             service: service.split('.')[1],
             service_data: { entity_id: id, ...serviceData },
         });
-        console.log(res);
+        // console.log(res);
         return res;
     }
 
-    async toggleEntity(id: string, enable: boolean): Promise<HaEntity | undefined> {
+    async toggleEntity(id: string, enable: boolean): Promise<Entity | undefined> {
```

Comment:
Instead of returning undefinded promises, I think we should use `throw Error(error)` and `.catch()` with promises.